hw/bsp/mimxrt1010_evk/board_config.h

```diff
@@ -1,22 +1,12 @@
 #ifndef _BOARD_CONFIG_H_
 #define _BOARD_CONFIG_H_
 
-// ===============================================================================
-//
-// TinyUF2 Configuration for NXP MIMXRT1010-EVK
-//
-// ===============================================================================
-
 // Basic Data
-// ==========
-
 #define VENDOR_NAME  "NXP"
 #define PRODUCT_NAME "MIMXRT1010-EVK"
-#define VOLUME_LABEL "UF2BOOT"
-#define INDEX_URL    ""
 #define BOARD_ID     "MIMXRT1010-EVK"
 
-// Set VID & PID for board in bootloader mode
+// VID & PID
```

Comment:
Not sure this comment adds much

Reply:
Because the board may (and probably will, in the general case) have a different VID:PID in run mode....this comment is to make that explicit.